accounts/users.go

```diff
@@ -20,54 +21,70 @@ func CheckPasswordHash(hash, password []byte) bool {
 	return err == nil
 }
 
+// ValidatePassword ensures a password meets minimum security requirements
+func ValidatePassword(password string) bool {
```

Comment:
Go question: So I read that functions that are exported and used elsewhere start with a capital letter, while those that are internal functions use camelcase; is that correct?